codegen/module.go

```diff
@@ -698,76 +698,76 @@ func (system *ModuleSystem) readInstance(
 	instanceDirectory string,
 ) (*ModuleInstance, error) {
 
-	jsonFileName := className + configSuffix
+	yamlFileName := className + configSuffix
```

Comment:
I think we should be looking for both `foo-config.yaml` and `foo-config.json`, preferring the `.yaml` file if it exists, otherwise falling back to `.json` for backwards compatibility.

Reply:
"configSuffix" is still "json". In the future diff, we will create a helper function to get the correct fileName for both json and yaml.